jr-extension-datetime/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.fasterxml.jackson.jr</groupId>
+        <artifactId>jackson-jr-parent</artifactId>
+        <version>2.17.0-rc1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jackson-jr-extension-datetime</artifactId>
+    <packaging>bundle</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.jr</groupId>
+            <artifactId>jackson-jr-objects</artifactId>
+        </dependency>
+    </dependencies>
+
```

Comment:
Ok, this needs bit more work: you can look at other `pom.xml`s like under `jr-retrofit2` or `jr-stree`. OSGi metadata is needed (for packaging as "bundle"), moditect generation.
And I think Gradle metadata.
I can help with these too.